Frontend/src/components/authInputField.js

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react';
+import {StyleSheet, Text, View, TextInput} from 'react-native';
+
+const AuthInputField = ({
+  title,
+  setData,
+  placeholderTextColor = '#6da7f2',
+  returnKeyType = 'next',
+  blurOnSubmit = false,
+  secureTextEntry = false,
+}) => {
+  return (
+    <View>
+      <Text style={styles.textBeforeTextInput}>{title}</Text>
+      <TextInput
+        style={styles.textInput}
+        onChangeText={(UserData) => setData(UserData)}
```

Comment:
onChangeText={setData}
It will call setData with a parameter so there is no need to do (userData) => setData(userData)